lineman/app/components/memberships_page/memberships_panel/memberships_panel.haml

```diff
@@ -0,0 +1,16 @@
+.blank
```

Comment:
No need to do this now.. but FYI .blank is a hack for a bug we're discovered when you try to put ng-if on the first element in a template. When there is no .ng-if  then you don't need .blank.. however this can't hurt.